lib/src/query_component.dart

```diff
@@ -45,35 +49,31 @@ class QueryComponent implements OnInit {
   final QueryService _queryService;
   final ExcelDart _excel;
 
-  QueryType _queryType;
+  List<QueryType> queryTypeChoices = QueryType.values;
+  QueryType queryType;
 
-  String advertiserId;
-  String insertionOrderId;
+  String advertiserId = '164337';
+  String mediaPlanId = '699530';
+  String insertionOrderId = '8127549';
   bool highlightUnderpacing = false;
 
-  // Radio button states with byAdvertiser selected as default.
-  RadioButtonState advertiserQuery = RadioButtonState(true, 'byAdvertiser');
-  RadioButtonState insertionOrderQuery = RadioButtonState(false, 'byIO');
-
   QueryComponent(this._queryService, this._excel);
 
   @override
   void ngOnInit() async => await _excel.loadOffice();
 
   void onClick() async {
-    // Determines the query type from radio buttons.
-    _queryType = advertiserQuery.checked
-        ? QueryType.byAdvertiser
-        : QueryType.byInsertionOrder;
-
     // Uses DV360 public APIs to fetch entity data.
     var insertionOrders = await _queryAndParseInsertionOrderEntityData();
```

Comment:
opt: add lhs data type. it's up to taste. Imo it's not obvious just by reading the method name (unlike a contructor for example) even though its inferable from the signature.